packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/LineComponent.java

```diff
@@ -107,6 +108,11 @@ private Element doRender(VariableManager lineVariableManager, String targetObjec
                                 .findFirst()
                                 .ifPresent(rowElementProps::height));
 
+        this.props.tableEvents().stream()
+                .filter(ResetTableRowsHeightEvent.class::isInstance)
+                .findFirst()
+                .ifPresent(iTableEvent -> rowElementProps.height(null));
```

Comment:
Set it back to the value provided by the specifier, you have an initialHeightProvider.